Files/Views/Pages/Properties.xaml

```diff
@@ -1,18 +1,26 @@
 ﻿<Page
     x:Class="Files.Views.Properties"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Media"
     d:DesignHeight="700"
     d:DesignWidth="400"
-    Background="{ThemeResource SolidBackgroundFillColorBaseBrush}"
+    Background="{ThemeResource SystemControlAcrylicWindowBrush}"
     KeyDown="Page_KeyDown"
     Loaded="Properties_Loaded"
     Loading="Page_Loading"
     Unloaded="Properties_Unloaded"
     mc:Ignorable="d">
-
+    <Page.Resources>
+        <muxc:AcrylicBrush
+            x:Key="SystemControlAcrylicWindowBrush"
+            BackgroundSource="HostBackdrop"
+            FallbackColor="{StaticResource SystemChromeMediumColor}"
+            TintColor="{StaticResource SystemChromeAltHighColor}"
```

Comment:
Should this use the new WinUI brushes we are using in other parts of the app?

Reply:
@yaichenbaum What do you mean?

Reply:
Something like `SolidBackgroundFillColorBase`.

Reply:
@yaichenbaum I think we should stick with these colors for consistency with other acrylic effects since they are the same as the default. I copied this brush directly from the v2 styles in the Microsoft.UI.Xaml repo, the only thing I changed was the opacity.

Reply:
One reason to change the brush would be to reduce how many brushes need to be changed in custom themes.

Reply:
Here's what it looks like
![image](https://user-images.githubusercontent.com/59544401/115800740-13ef4c80-a390-11eb-95c7-576d457b6cb2.png)